hs_restclient/__init__.py

```diff
@@ -1088,8 +1088,91 @@ def getUserInfo(self):
 
         return r.json()
 
+    def getTicket(self, pid, pathname=None, mode='read'):
+        """Get a ticket for a specific *pathname* for a given resource
 
-class AbstractHydroShareAuth(object): pass
+        :param pid: The HydroShare ID of the resource for which folder is to be deleted
```

Comment:
A ticket at this point can read more or less anything, including a bag, folder, or file. There is no current support for deleting a folder, though I anticipate supporting that. See below for full details of API. 

I don't know how the line above got deleted, though. That is an error. Will fix.